modules/msk/genomenexus/vcf2maf/main.nf

```diff
@@ -0,0 +1,45 @@
+process GENOMENEXUS_VCF2MAF {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'ghcr.io/msk-access/genomenexus:latest':
+        'ghcr.io/msk-access/genomenexus:latest' }"
+
+    input:
+    tuple val(meta), path(vcf)
+
+    output:
+    tuple val(meta), path("vcf2maf_output/${meta.id}.maf"), emit: maf
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    
+
+    """
+    python3 /annotation-tools/vcf2maf.py -i ${vcf} ${args}
+    sleep 3
+
+    echo '"${task.process}": vcf2maf.py --help' > versions.yml
```

Comment:
The versioning output should follow the nf-core style: 

```
cat <<-END_VERSIONS > versions.yml
"${task.process}":
    fastqc: \$( fastqc --version | sed -e "s/FastQC v//g" )
    samtools: \$( samtools --version |& sed '1!d ; s/samtools //' )
END_VERSION
```
another example: https://github.com/mskcc-omics-workflows/modules/blob/develop/modules/msk/snppileup/main.nf#L36-L42

Reply:
hard code in the versioning if not available